ArchLinux/PKGBUILD

```diff
@@ -29,7 +29,9 @@ build() {
    cd bin
    cmake .. -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr \
-   -DCMAKE_INSTALL_LIBDIR=lib
+   -DCMAKE_INSTALL_LIBDIR=lib \
+   -DCMAKE_C_FLAGS="-Wno-format -Wno-error=format-security" \
+   -DCMAKE_CXX_FLAGS="-Wno-format -Wno-error=format-security"
```

Comment:
Let's move these flags into root CMakeLists.txt.

Somewhere in future we will remove them. (#713)

Reply:
Ok then

Reply:
@NeoTheFox, NO, please don't open new pull request. Do it in this pull request.

Reply:
If you wish